cmd/tar2ext4/tar2ext4.go

```diff
@@ -1,63 +1,317 @@
 package main
 
 import (
-	"flag"
+	"context"
 	"fmt"
 	"io"
 	"os"
 
 	"github.com/Microsoft/hcsshim/ext4/tar2ext4"
+	"github.com/Microsoft/hcsshim/internal/log"
+	"github.com/urfave/cli"
```

Comment:
it may be worthwhile to just switch to v2 if we're adding this to PR from scratch
(https://cli.urfave.org/migrate-v1-to-v2/)